PWGGA/GammaConv/AliAnalysisTaskGammaCalo.cxx

```diff
@@ -901,9 +902,12 @@ void AliAnalysisTaskGammaCalo::UserCreateOutputObjects(){
   Float_t maxClusterPt        = 50;
   Int_t nBinsMinv             = 800;
   Float_t maxMinv             = 0.8;
+  float minRes                = -1.f;
+  float maxRes                = +5.f; 
```

Comment:
are the -1.f and +5.f deliberate? I have not seen it done that way yet.

Reply:
From my understanding any floating point number like 1. would first be assumed to be a double by the compiler, so writing 1.f makes sure it is a float and there is no extra conversion from double to float happening then.